### DISCUSSION TITLE
expo-auto-navigation

### DISCUSSION THREAD

**Discussion by redbar0n** [expo/router#17] [Q&A] [answered]
@EvanBacon Just to confirm, this Expo Router /  expo-router is the succession of your previous experiments/proof-of-concepts [expo-auto-navigation](https://github.com/EvanBacon/expo-auto-navigation)  and [expo-auto-navigation-webpack](https://github.com/EvanBacon/expo-auto-navigation-webpack), right?

**Comment by EvanBacon** [comment, marked as answer]
Correct, I'll update the readmes.
